schemas/digital-outcomes-and-services-2.declaration.json

```diff
@@ -0,0 +1,69 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "allOf": [
+        {"$ref": "#/definitions/notDefiniteFail"},
+        {
+            "properties": {
+                "taxEvasion": {"enum": [false]},
+                "environmentalSocialLabourLaw": {"enum": [false]},
+                "bankrupt": {"enum": [false]},
+                "graveProfessionalMisconduct": {"enum": [false]},
+                "distortingCompetition": {"enum": [false]},
+                "conflictOfInterest": {"enum": [false]},
+                "distortedCompetition": {"enum": [false]},
+                "significantOrPersistentDeficiencies": {"enum": [false]},
+                "seriousMisrepresentation": {"enum": [false]},
+                "witheldSupportingDocuments": {"enum": [false]},
+                "influencedContractingAuthority": {"enum": [false]},
+                "confidentialInformation": {"enum": [false]},
+                "misleadingInformation": {"enum": [false]},
+                "unspentTaxConvictions": {"enum": [false]},
+                "GAAR": {"enum": [false]}
+            }
+        }
+    ],
+    "definitions": {
+        "notDefiniteFail": {
+            "$schema": "http://json-schema.org/draft-04/schema#",
+            "type": "object",
+            "properties": {
+                "status": {"enum": ["complete"]},
+                "termsOfParticipation": {"enum": [true]},
+                "termsAndConditions": {"enum": [true]},
+                "10WorkingDays": {"enum": [true]},
+                "informationChanges": {"enum": [true]},
+                "requisiteAuthority": {"enum": [true]},
+                "accurateInformation": {"enum": [true]},
+                "unfairCompetition": {"enum": [true]},
+                "readUnderstoodGuidance": {"enum": [true]},
+                "understandTool": {"enum": [true]},
+                "understandHowToAskQuestions": {"enum": [true]},
+                "proofOfClaims": {"enum": [true]},
+                "skillsAndResources": {"enum": [true]},
+                "accuratelyDescribed": {"enum": [true]},
+                "employersInsurance": {
+                    "type": "string",
+                    "pattern": "^(Yes |Not applicable )"
```

Comment:
We probably don't need the trailing spaces after the values here?  Might make it a bit more readable I guess so fine either way.  I suppose once we auto-generate this stuff we'll probably end up with the whole long value here :/